tests/api/security.test.js

```diff
@@ -2360,6 +2360,113 @@ module.exports = function() {
       });
     });
 
+    describe('api/brokerlogs', function() {
+      describe('List all broker logs (GET)', function () {
+
+        it('Admin should be authorized', function(done) {
+          nano.request(sails.hooks.http.app)
+            .get('/api/brokerlogs')
+            .set(nano.adminLogin())
+            .expect(200)
+            .end(done);
+        });
+
+        it('Regular users should be unauthorized', function(done) {
+          nano.request(sails.hooks.http.app)
+            .get('/api/brokerlogs')
+            .set('Authorization', 'Bearer ' + token)
+            .expect(403)
+            .end(done);
+        });
+
+        it('Unauthenticate users should be unauthorized', function(done) {
+          nano.request(sails.hooks.http.app)
+            .get('/api/brokerlogs')
+            .expect(401)
+            .end(done);
+        });
+      });
+
+      describe('Create a broker log', function () {
+
+        it('Admin should be authorized', function(done) {
```

Comment:
Only internal logic should ever update information on this table. Only `find` and `findOne` should be accessible for admins.